pareto/strategic_water_management/run_strategic_model.py

```diff
@@ -143,6 +148,7 @@
         "objective": Objectives.cost,
         "pipeline_cost": PipelineCost.distance_based,
         "pipeline_capacity": PipelineCapacity.input,
+        "hydraulics": Hydraulics.post_process,
```

Comment:
```suggestion
        "hydraulics": Hydraulics.false,
```
I think we want the default hydraulics setting to be `false`.

Reply:
done